components/raftstore/src/store/worker/pd.rs

```diff
@@ -992,7 +995,22 @@ where
                 );
             }
         };
-        spawn_local(f);
+
+        #[cfg(feature = "failpoints")]
+        let delay = (|| {
+            fail_point!("delay_update_max_ts", |_| true);
+            false
+        })();
+        #[cfg(not(feature = "failpoints"))]
+        let delay = false;
+
+        if delay {
+            info!("[failpoint] delay update max ts for 1s"; "region_id" => region_id);
```

Comment:
Can it print a lot? What's the usage of this failpoint, long time tests?

Reply:
I think this will be triggered when region split ,merge or leader transfer happens.
This failpoint is expected to delays updating the max_ts in these cases, which makes MaxTsNotSynced error more likely to happen.

Reply:
I concern this one.
```rust
            fail_point!("async_commit_1pc_force_fallback", |_| {
                info!("[failpoint] injected fallback for async commit/1pc transaction"; "start_ts" => start_ts);
                true
            });
            false
```

Reply:
Do  you think it's unnecessary? I don't have any strong opinion about whether it should be removed, just thinking it might help us debugging when something went  wrong during tests, since it tells us what failpoint the transaction has encountered. There's also similar logs in many of TiDB's failpoints. 😕